app/styles/styles.js

```diff
@@ -0,0 +1,95 @@
+import { StyleSheet } from "react-native";
+
+export const layoutStyles = StyleSheet.create({
+  backgroundLayout: {
+    flex: 1,
+    backgroundColor: "#383838"
+  } 
+})
+export const loginOptionsStyles = StyleSheet.create({
+  loginLayoutCentering: {
+    justifyContent: "center",
+    alignItems: "center"
+  },
+
+  contentContainer: {
+    alignItems: "center",
+    justifyContent: "space-between",
+    width: 270,
+    height:461,
+  },
+  
+  circleElementTitleContainer: {
+    width: 174,
+    height: 193,
+    alignItems: "center",
+    justifyContent: "space-between"
+  },
+
+  circleElement: {
+    width: 96,
+    height: 96,
+    borderRadius: 96/2,
+    backgroundColor: "#C4C4C4",
+  },
+
+  Title: {
+   width: 174,
+   height: 84,
+   justifyContent: "center",
+   alignItems: "center",
+   fontSize: 72,
+   fontStyle: "Italic",
```

Comment:
I was getting an error with the capitalized 'Italic' here. Says it needs to be lowercase.